src/StrawberryShake/CodeGeneration/test/CodeGeneration.Tests/Utilities/__snapshots__/QueryDocumentRewriterTests.GetReturnTypeName.snap

```diff
@@ -2,12 +2,6 @@
   hero(episode: NEW_HOPE) {
     __typename
     ... Characters
-    ... on Droid {
-      id
-    }
-    ... on Human {
-      id
-    }
```

Comment:
Is this related to this PR?

Reply:
not really.

Reply:
Must be from a merge ... will have a look at this tomorrow... well spotted 👍

Reply:
OK... this is from the merge the query is ok